src/ui/reportView/callReport.ts

```diff
@@ -18,9 +18,16 @@ interface htmlMetaData {
 	searchDir: string;
 }
 
+interface visualizeResult {
```

Comment:
`visualizeResult` is too similar to `visualizeOutput` by name but they talk about different things. I'd rename visualizeResult to `ReportMetadata` I think.

Reply:
Agree, changed to `reportMetadata`.